modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue

```diff
@@ -1,22 +1,37 @@
 <template>
   <div class="apos-field__outer">
     <component :is="wrapEl" :class="classList">
-      <!-- TODO i18n -->
-      <component
-        v-if="field.label" :class="{'apos-sr-only': field.hideLabel }"
-        class="apos-field-label"
-        :is="labelEl" :for="uid"
-      >
-        {{ field.label }}
-        <span v-if="field.required" class="apos-field-required">
-          *
-        </span>
-      </component>
-      <!-- TODO i18n -->
-      <p v-if="field.help" class="apos-field-help">
-        {{ field.help }}
-      </p>
-      <slot name="additional" />
+      <div class="apos-field-info">
+        <!-- TODO i18n -->
+        <component
+          v-if="field.label" :class="{'apos-sr-only': field.hideLabel }"
+          class="apos-field-label"
+          :is="labelEl" :for="uid"
+        >
+          {{ field.label }}
+          <span v-if="field.required" class="apos-field-required">
+            *
+          </span>
+          <span
+            v-if="field.help && displayOptions.helpTooltip"
+            class="apos-field-help-tooltip"
```

Comment:
This component seems to be totally abandoning BEM.

Reply:
re-bem